ui/lib/pki/addon/components/parsed-certificate-info-rows.js

```diff
@@ -0,0 +1,32 @@
+import Component from '@glimmer/component';
+import { parsedParameterKeys } from 'vault/utils/parse-pki-cert-oids';
+
```

Comment:
Can we add documentation here?

Reply:
If it's not standard practice in the PKI addon components, then ignore.

Reply:
Sure, I can add the JSDoc formatted notes